src/core/utils/compatibility.js

```diff
@@ -139,7 +140,10 @@ export function isCompatibleWithUserAgent({
   // Fenix does not support add-ons (yet?).
   // See: https://github.com/mozilla-mobile/fenix/issues/1134
   // See also: https://github.com/mozilla/addons-frontend/issues/7963
-  if (os.name === 'Android' && mozCompare(browser.version, '69.0') >= 0) {
+  if (
+    os.name === USER_AGENT_OS_ANDROID &&
```

Comment:
mmh so we have `OS_ANDROID` and `USER_AGENT_OS_ANDROID`? I dont think we need both, do we?

Reply:
Ah, but `OS_ANDROID` is `android`, whereas `USER_AGENT_OS_ANDROID` is `Android`, so we do actually need both. It's an ugly situation, but not one that I think is worth dealing with at this point.

Reply:
I'd suggest creating a code quality issue for this ugly situation.

Reply:
Because we don't want stalebot to get lonely? ;-)

Reply:
I think the only way to properly "fix" it would be to adjust our code that processes user agent data to normalize `Android` to `android`, and that might not even be desirable, so I have my doubts about whether this will ever get addressed, but sure, I can open a code quality issue.

Reply:
I opened https://github.com/mozilla/addons-frontend/issues/8925